### ISSUE TITLE
Please provide better instructions on how to add/install custom flavor

### ISSUE THREAD

**Issue by kristianmandrup** [cyclejs-community/create-cycle-app#89]
It is not really clear how to add/install a custom flavor. I asked same question [here](https://github.com/Widdershin/cycle-scripts-widdershin/issues/1) and looked at [create-cycle-app-flavors](https://github.com/cyclejs-community/create-cycle-app-flavors) and [cycle-scripts](https://github.com/cyclejs-community/create-cycle-app/tree/master/packages/cycle-scripts) but still not clear...

I wonder if your script can somehow find it by `package.json` `keyword` entry (like I've seen in Ember CLI f.ex) or if I have to install my flavor as a global module or what!?

Please make it more clear. Thanks

**Comment by nickbalestra** [comment]
In the documentation there is a section dedicated on [custom flavors](https://github.com/cyclejs-community/create-cycle-app#custom-flavour). 

where it says:
-  Flavors can be published to npm, or being used locally via the create-cycle-app CLI.
- When creating a project, you can inform which flavor you want to use with the --flavor flag

and some example are given:

```
$ create-cycle-app my-app --flavor cycle-scripts-es-webpack

$ create-cycle-app my-app --flavor cycle-scripts-es-webpack@x.y.z

$ create-cycle-app my-app --flavor ./relative/path/to/cycle-scripts-es-webpack
```

I guess this is not clear enough, what from that section was not clear for you Kristian?

**Comment by nickbalestra** [comment]
One thing I'm noticing is also that those example of custom flavors are not supported anymore, so it should perhaps use cycle-scripts-one-fits-all instead

**Comment by kristianmandrup** [comment]
Yes, unfortunately it is all quite a mess... As usual. Web dev moves too fast.
My attempt at writing a clearer description can be seen [here](https://github.com/kristianmandrup/create-cycle-app-flavors)

But nowhere have I seen any docs on how to fine-tune the scripts, such as babel compile options etc. I'm so exhausted from having to spend 80% of my dev time trying to figure things out instead of getting things done. Maybe I have overlooked, hard to read through 10s-100s of docs pages, repos/code etc. every day in order to make a step forward. Easy to miss sth. when you are frustrated.

Also unfortunate that most of the resources on awesome-cycle are outdated and pretty useless except as ideas/reference points. We need more curation of which things are up to date... 
Cheers!

**Comment by kristianmandrup** [comment]
Better remove the flavors that are obsolete ;) Too many dead ends...

**Comment by geovanisouza92** [comment]
@kristianmandrup You are welcome to suggest anything on any Cycle.js-related repo/project. Everyone on the community accept suggestions and discussion.

However, it's a small community, and we need help from everyone that is available to help.

Feel free to propose content curation to the community. :)

**Comment by kristianmandrup** [comment]
@geovanisouza92 My main concern is that it is impossible to see which projects are recent, popular or outdated... would be nice with some form of better structuring than simply by category. 
Perhaps we need a new site/app to better facilitate that?

**Comment by geovanisouza92** [comment]
Perhaps. Try to share your thoughts on gitter. I'm sure that we all will find a way :)
